core/cli/commands/database.py

```
@@ -3196,3 +3196,283 @@ def link_code_smell(
+                        # Single AQL call per collection: iterate precomputed list
+                        result = client.query(
+                            f"""
+                            FOR u IN @updates
+                                UPDATE {{_key: u._key}} WITH {{text: u.text}} IN {col_name}
+                                COLLECT WITH COUNT INTO n
+                                RETURN n
+                            """,
+                            bind_vars={"updates": updates},
+                        )
+                        updated = result[0] if result else len(updates)
+                    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)